vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -44,7 +44,7 @@ dat1 <- checkMissingsByValues(dat1, missingValues = -50:-99)
 # -----------------------------------------------------------------------------
 ## Recode IDs
 The first step is to create the path for the template.
-```{r IDrecode}
+```{r IDrecode, eval = FALSE}
```

Comment:
👍  but this triggers problems in later code chunks. Let's discuss this after lunch.

Reply:
true, next time I'll knit it first!